docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+            "ultra_veto_enabled": 1,
+            "max_transaction_lifetime": 3600,
+            "deferred_trx_expiration_window": 600,
+            "max_transaction_delay": 3888000,
+            "max_inline_action_size": 524287,
+            "max_inline_action_depth": 10,
+            "max_authority_depth": 10
+          }
+        }
+merger:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    persistentVolumeClaimEnabled: false
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+relayer:
+  replicaCount: 1
+  deployment:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    relayerSource2: ""
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+substreams-tier1:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    substreamsStateStoreUrl: "file:///storage/state-store"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+substreams-tier2:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+    substreamsStateStoreUrl: "file:///storage/state-store"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+firehose:
+  replicaCount: 1
+  statefulset:
+    commonOneBlockStoreUrl: "file:///storage/one-blocks"
+    commonMergedBlocksStoreUrl: "file:///storage/merged-blocks"
+    commonForkedBlocksStoreUrl: "file:///storage/forked-blocks"
+  # enabled local storage to replace gloud storage
+  localVolume:
+    enabled: true
+
+```
+
+Some important configurations you need to know:
+
+- `statefulset: enabledPVC = false`: This will disable the persistent volume use for Ultra's deployment
+- `localVolume: enabled = true`: This will enable local storage to replace cloud storage, this will make use of `minikube` persist file store. You can read more here [Persistent Volumes](https://minikube.sigs.k8s.io/docs/handbook/persistent_volumes/).
+- `p2p-peer-address = producer-0.producer.default.svc.cluster.local:9876`: you can see why I mentioned the importance of this config above. If you use a different name/namespace please update the according to the format mentioned in [DNS Pod](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pods) or simply use get pod IP and replace it here.
+
+The final step is to install `firehose-antelope` chart
+
+```sh
+helm install firehose-antelope ./firehose-antelope -f ./values.yaml
+```
+If the deployment has no error, you should see this
```

Comment:
Here we validate firehose antelope pods are running, but are there any other options to validate the pods are in fact working properly? For example, running `get info` from `producer-0` node.

Reply:
Well, actually if some pod is not working well, the `READY` status will be 0/1. (exclude `firehose`)
Also, the end goal is to have substreams sink or CLI that work with this one, so that will validate these deployments are working fine.